Imager.js

```diff
@@ -142,30 +143,36 @@
           });
         }
 
-        if (elements) {
-            this.divs = applyEach(elements, returnFn);
-            this.selector = null;
-        }
-        else {
-            this.divs = applyEach(doc.querySelectorAll(this.selector), returnFn);
-        }
-
+        this.divs = [];
+        this.add(elements || this.selector);
         this.ready(opts.onReady);
-        this.changeDivsToEmptyImages(this.divs);
 
-        setTimeout(function(){
+        setTimeout(function () {
             self.init();
         }, 0);
-    }
+    };
+
+    Imager.prototype.add = function (elementsOrSelector) {
+
+        elementsOrSelector = elementsOrSelector || this.selector;
```

Comment:
Because of the *real change no1*, `this.selector` can be null. To be honest I would allow an empty call like `imgr.add()`: it would rematch existing responsive images handled by Imager as no deduplication is involved (which is another problem, but I don't want that to be eased up).

What do you think?